knowledge-base/aliyun/acr-login/action-security.yml

```diff
@@ -0,0 +1,13 @@
+name: 'ACR-Login action'
+outbound-endpoints:
+  - fqdn: 'cr.*.aliyuncs.com'
+    port: 443
+    reason: get api endpoints
```

Comment:
Thanks for the PR! 

I ran the tests and it failed with this:
>Reason must start with 'to '. It is currently get api endpoints in action-security.yml at knowledge-base/aliyun/acr-login/action-security.yml. 

This will apply to all the `reasons`